manifests/init.pp

```diff
@@ -990,7 +990,7 @@
             require => Package[$my_package_name],
           }
 
-          file { 'pam_common_noninteractive_session':
+          file { 'pam_common_session_noninteractive':
```

Comment:
That's part of the "Standardize naming" commit. It would remove some special cases from the spec tests, but it not needed.